src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -142,12 +142,7 @@ def _generate_parameter_files(
 class EnKFMain:
     def __init__(self, config: "ResConfig", read_only: bool = False):
         self.res_config = config
-        self.update_snapshots = {}
```

Comment:
Was under the impression this was used here: https://github.com/equinor/ert/blob/d9fd76cf2782531dc01a8e3e1d0b87586661d84c/src/ert/shared/storage/extraction.py#L180, but maybe it is outdated

Reply:
That `ert` refers to `LibresFacade` I think, where `update_snapshot` returns `self._es_udpate.update_snapshots`.